<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackageSelector</class>
 <widget class="QDialog" name="PackageSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Package Selector</string>
  </property>
  <widget class="QListWidget" name="elementListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="packageListWidget">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>220</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="elementLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Element</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="packageLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Package</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
